src/greentea/gtea/greentea_hooks.py

```diff
@@ -6,7 +6,8 @@
 import os
 import re
 import json
-from subprocess import Popen, PIPE
+
+from subprocess import run
```

Comment:
Usually the style is to `import subprocess` and then call `subprocess.run` as it's then explicit what the `run` function is doing. The function name itself is rather generic, so using the namespaced form at the call site is better in this case.